android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -1038,7 +1050,9 @@ public ProjectConfig getProjectConfigPolling(String url)
 		public int getBatteryChargeStatus() throws RemoteException{
 			try {
 				return getDeviceStatus().getStatus().battery_charge_pct;
-			} catch (Exception e) {}
+			} catch (Exception e) {
+				if(Logging.ERROR) Log.e(Logging.TAG,"Monitor.getBatteryChargeStatus error: ",e);
```

Comment:
For consistency:  `Monitor.IMonitor.Stub.getBatteryChargeStatus`

Reply:
Fixed